core/src/main/java/org/kiwix/kiwixmobile/core/bookmark/BookmarksContract.kt

```diff
@@ -15,24 +15,27 @@
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  *
  */
+package org.kiwix.kiwixmobile.core.bookmark
 
-package org.kiwix.kiwixmobile.core.bookmark;
+import org.kiwix.kiwixmobile.core.base.BaseContract
+import org.kiwix.kiwixmobile.core.base.BaseContract.Presenter
+import org.kiwix.kiwixmobile.core.base.BaseContract.View
 
-import java.util.List;
-import org.kiwix.kiwixmobile.core.base.BaseContract;
-
-interface BookmarksContract {
-  interface View extends BaseContract.View<Presenter> {
-    void updateBookmarksList(List<BookmarkItem> bookmarks);
-
-    void notifyBookmarksListFiltered(List<BookmarkItem> bookmarks);
+internal interface BookmarksContract {
+  interface View :
+    BaseContract.View<Presenter?> {
+    fun updateBookmarksList(bookmarks: List<BookmarkItem?>?)
```

Comment:
are all these parameters nullable? I really hope they are not nullable

Reply:
No they aren't. I should have been more careful. Fixed now :)